src/components/DateInput.jsx

```diff
@@ -217,7 +220,7 @@ class DateInput extends React.PureComponent {
             focused && styles.DateInput_input__focused,
             disabled && styles.DateInput_input__disabled,
           )}
-          aria-label={placeholder}
+          aria-label={ariaLabel}
```

Comment:
we'd need to retain the `ariaLabel || placeholder` fallback here, otherwise this is a breaking change.

Reply:
Ah, got it. Didn't think about the breaking nature of this. I'll want some way to not have any `aria-label` field, so what do you think about having a ternary like: `ariaLabel === undefined ? placeholder : null`

Reply:
I’d do `: undefined, but that looks fine to me!

Reply:
Yep, idk why I love `null` so much. I'll change it to `undefined`.

Reply:
Yeah, I goofed the example a bit. I used `ariaLabel` as the second case in the ternary, it shouldn't have been `null`.